src/MeetingsSDKAdapter.js

```diff
@@ -0,0 +1,171 @@
+import {MeetingsAdapter} from '@webex/component-adapter-interfaces';
+import {concat, from, fromEvent, Observable} from 'rxjs';
+import {filter, finalize, map, publishReplay, refCount} from 'rxjs/operators';
+
+const MEDIA_TYPE_LOCAL = 'local';
+const EVENT_MEDIA_READY = 'media:ready';
+
+/**
+ * The `MeetingsSDKAdapter` is an implementation of the `MeetingsAdapter` interface.
+ * This adapter utilizes the Webex JS SDK to create and join webex meetings.
+ *
+ * @class MeetingsSDKAdapter
+ * @extends {MeetingsAdapter}
+ */
+export default class MeetingsSDKAdapter extends MeetingsAdapter {
+  constructor(datasource) {
+    super(datasource);
+    this.getMeetingsObservables = {};
```

Comment:
Let's make this singular like `getPersonObservables` and `getRoomObservables`

```suggestion
    this.getMeetingObservables = {};
```